### ISSUE TITLE
Forced online license activation

### ISSUE THREAD

**Issue by khromov** [vslavik/poedit#421]
As a long-term user and customer of Poedit, I'm very critical of the fact that Poedit now requires online activation. This is hostile DRM that should be encouraged or supported. I urge you to reconsider @vslavik

**Comment by vslavik** [comment]
This is not an appropriate place — please use help@poedit.net for Pro-related stuff. First, this is the bug tracker for the Open Source version of Poedit, second, it is for *tracking bugs*, not exaggerated ideological arguments.

You will note that v1 did not use activation (because of, yes, the same ideological reasons). Unless you think I'm an irrational moron (thanks for that), you should suspect, if not be reasonably certain, that there might be some *reasons* for the change. And oh boy, indeed there are.

In theory, I agree with you — again, see v1 (or my track record in OSS). In practice, out there in the *real* world, some people are horrible and don't hesitate to violate the contracts they enter. Hence activation, which I'm sure you're well aware is a common practice nowadays.

Contrary to your mischaracterization of it as "hostile", Poedit's activation is minimally invasive, only requiring being online *at the very moment you purchased a license online*. And that's not to mention that major Pro functionality requires internet access *anyway*, so requiring it more frequently wouldn't be a big deal either — but I didn't do it that way, because *I* didn't want activation to be invasive. There are more aspects of it that I won't discuss here that were driven purely by the desire to be non-invasive and non-annoying (and not like e.g. the somewhat popular strict DevMate activation).

In short: I consider it necessary for the time being. I *also* consider it necessary to tweak it, and continue to tweak it, to ensure it won't be annoying in practice. But fix humanity to be 100% decent and I'll gladly roll it back.
